kong/plugins/kubernetes-sidecar-injector/handler.lua

```diff
@@ -1,10 +1,7 @@
-local BasePlugin = require "kong.plugins.base_plugin"
-
-local Handler = BasePlugin:extend()
-
-Handler.VERSION = "0.1.0"
-
--- priority doesn't matter, just need to pick something unique for kong tests
-Handler.PRIORITY = 1006
+local Handler = {
+  VERSION = "0.2.0",
+  -- priority doesn't matter, just need to pick something unique for kong tests
+  PRIORITY = 1006,
+}
```

Comment:
Is this code needed?
We ship the sidecar injector with Kong.

Reply:
This code *is* what gets shipped with kong.

Reply:
I see.
It feels odd that we are updating this code in this PR then. Please open a separate PR for this change.

Reply:
@hutchic Please remove this change and we are good to merge.

Reply:
this matches master and I squashed the commits. I think it only shows up in the github diff because of the comment